src/Container/ApplicationFactory.php

```diff
@@ -77,19 +63,13 @@ public function __invoke(ContainerInterface $container)
             ? $container->get(RouterInterface::class)
             : new FastRouteRouter();
 
-        $finalHandler = ! empty($config['zend-expressive']['raise_throwables'])
-            ? $this->marshalNoopFinalHandler($container)
-            : $this->marshalLegacyFinalHandler($container, $config);
+        $delegate = $this->marshalNoopFinalHandler($container);
 
         $emitter = $container->has(EmitterInterface::class)
             ? $container->get(EmitterInterface::class)
             : null;
 
-        $app = new Application($router, $container, $finalHandler, $emitter);
-
-        if (! empty($config['zend-expressive']['raise_throwables'])) {
```

Comment:
`raiseThrowables` has no semantic meaning in Stratigility 2.0; it's just how the `Next` implementation works.